docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -23,6 +32,16 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+The ``connection-url`` defines the connection information and parameters to pass
+to the PostgreSQL JDBC driver. The supported parameters for the URL are
+available in the `PostgreSQL JDBC driver documentation
+<https://jdbc.postgresql.org/documentation/head/connect.html>`_.
```

Comment:
There may be parameters that make the driver behave in a way that causes the connector to work incorrectly.

(For _example_, `stringtype`, `autosave`, `binaryTransfer`, `preferQueryMode`, `disableColumnSanitiser`, `currentSchema`, `reWriteBatchedInserts`, `escapeSyntaxCallMode`)

Reply:
okay .. so should I add a warning about these? Or do we specify which parameters are allowed? Do you have any details there?

Reply:
> about these? 

The list was not meant to be exhaustive, and it was based on skimming the page.
no, enumerating the problematic ones doesn't IMO make sense, and we don't have the bandwidth to keep the list up to date.


> Or do we specify which parameters are allowed?

I don't have a bandwidth to test them

maybe we write something like "if you set any properties the connector may or may not work correctly"

or maybe we simply don't link to that page at all?

Reply:
linking to the page is necessary so people can find info on how to set up TLS/SSL and such if needed

Reply:
I updated now .. having the link is requested by users pretty regularly btw.